packages/plans-grid-next/src/components/test/tooltip.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import Tooltip from '../shared/tooltip';
+
+jest.mock( '@automattic/components', () => ( {
+	Tooltip: ( { children, isVisible } ) => ( isVisible ? children : null ),
+} ) );
```

Comment:
Can we _not_ mock this?

Reply:
Sure, updated to use a different approach: [6e2026c](https://github.com/Automattic/wp-calypso/pull/93518/commits/6e2026c6a3748465627d8d99bad74c11c18daab9)